core/src/main/java/org/elasticsearch/node/Node.java

```diff
@@ -739,6 +746,13 @@ BigArrays createBigArrays(Settings settings, CircuitBreakerService circuitBreake
     }
 
     /**
+     * Build the {@link SearchModule}. Overridden by tests.
+     */
+    protected SearchModule createSearchModule(List<SearchPlugin> plugins) {
```

Comment:
`SearchModule` creates the `SearchService` though. It'd be weird for `SearchModule` to delegate back out to `Module`....